src/main/resources/schemas/exchange/organizationDirectory.xsd

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+           xmlns:tns="urn:gov:hhs:fha:nhinc:exchange:directory"
+           targetNamespace="urn:gov:hhs:fha:nhinc:exchange:directory"
+           elementFormDefault="qualified">
+
+  <xsd:complexType name="organization">
+    <xsd:sequence>
+      <xsd:element name="name" type="xsd:string"/>
+      <xsd:element name="hcid" type="xsd:string"/>
+      <xsd:element name="status" type="xsd:string"/> <!-- for example, active -->
+      <xsd:element name="contact" type="tns:contactType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="address" type="tns:addressType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="partOf" type="tns:partOfType" minOccurs="0"/>
+      <xsd:element name="endpoint" type="tns:endpointType"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="contactType">
+    <xsd:sequence>
+      <xsd:element name="use" type="xsd:string" minOccurs="0"/> <!--techincal, financial -->
+      <xsd:element name="givenName" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="familyName" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="phone" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="address" type="tns:addressType" minOccurs="0"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="addressType">
+    <xsd:sequence>
+      <xsd:element name="use"  type="xsd:string" minOccurs="0"/> <!-- for example, work -->
```

Comment:
I was trying to give an example what is expected in this field. I will make my comments more clear.